<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIR Clients Integrations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>HL7</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLineEdit" name="leHL7IpAddress">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pbHL7Connect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="sbHL7Port">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lHL7SocketStatus">
            <property name="text">
             <string>&lt;span style=&quot; font-weight:700; color:#ff0000;&quot;&gt;Server disconnected.&lt;/span&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl7MessageSection">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lHL7MsgToSend">
              <property name="text">
               <string>Message HL7 to send</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teHL7MsgToSend"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbHL7SendMessage">
              <property name="text">
               <string>Send &gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lHL7MessageReceived">
              <property name="text">
               <string>Message received</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teHL7MessageReceived"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lHL7QuickOperations">
              <property name="text">
               <string>&lt;b&gt;Quick operations&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbHL7CreatePatient">
              <property name="text">
               <string>Create a new patient</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbHL7ReadSession">
              <property name="text">
               <string>Consult a session</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Exchange</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="2">
           <widget class="QComboBox" name="cbWXPMIRExecutable"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="leWXPMIRAppPath"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lWXPMIRDirectoryPath">
            <property name="text">
             <string>Directory of MIR application</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lWXPMIRExecutablePath">
            <property name="text">
             <string>MIR executable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pbWXPBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="cbWXPSelectOS">
            <item>
             <property name="text">
              <string>Windows</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MacOS</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Content of the Patient.SRV file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPPatientSRVContent"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pbWXPSendPatient">
                <property name="text">
                 <string>Send a patient</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbWXPReadPatientSession">
                <property name="text">
                 <string>Read a session</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit_4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="pbWXPCreatePatient">
              <property name="text">
               <string>Create a new patient</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbWXPCreatePatientSession">
              <property name="text">
               <string>Open a session</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>6</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Results.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="lWXPResultsWspFile">
              <property name="text">
               <string>Results.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResultsWspFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Results2ndBest.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="lWXPResults2ndBestFile">
              <property name="text">
               <string>Results2ndBest.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResults2ndBestFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Results3rdBest.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="lWXPResults3rdBestFile">
              <property name="text">
               <string>Results3rdBest.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResults3rdBestFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>ResultsPostBd.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="lWXPResultsPostBdFile">
              <property name="text">
               <string>ResultsPostBd.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResultsPostBdFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_8">
           <attribute name="title">
            <string>ResultsOxi.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="lWXPResultsOxiFile">
              <property name="text">
               <string>ResultsOxi.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResultsOxiFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_9">
           <attribute name="title">
            <string>SpiroResults.wsp</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>PDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leWXPResultsSpiroResultsPdf"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="lWXPResultsSpiroResultsFile">
              <property name="text">
               <string>SpiroResults.wsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="teWXPResultsSpiroResultsFile"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Curves</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="3">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="pbWXPCurvesFVC">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLineEdit" name="leWXPCurvesFVCVT"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="leWXPCurvesFVC"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>VC.jpg</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QPushButton" name="pbWXPCurvesFVCVT">
                <property name="text">
                 <string>Open File</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>FVC.jpg</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>FVC_VT.jpg</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>FVC_POST.jpg</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="leWXPCurvesVC"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="leWXPCurvesFVCPOST"/>
              </item>
              <item row="1" column="5">
               <widget class="QLineEdit" name="leWXPCurvesMVV"/>
              </item>
              <item row="2" column="5">
               <widget class="QLineEdit" name="leWXPCurvesOXI"/>
              </item>
              <item row="1" column="4">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>MVV.jpg</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>OXI.jpg</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="pbWXPCurvesVC">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="pbWXPCurvesFVCPOST">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QPushButton" name="pbWXPCurvesMVV">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
              <item row="2" column="6">
               <widget class="QPushButton" name="pbWXPCurvesOXI">
                <property name="text">
                 <string>Open file</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>GDT</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
